### PR TITLE
add dapplooker

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/282

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/integrations/analytics/dapplooker.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+随后，系统将提示您输入您项目和合约的详情：
+
+1. 输入您的项目名称
+
+2. 输入您项目运行所在的网络。网络可以选择**Moonbeam**、**Moonriver**或**Moonbase Alpha**
+
+3. 如果您有多个相同合约的实例，你可以启动按钮
+
+4. 输入您的合约地址。如果合约地址已在[Moonscan](https://moonscan.io/){target=_blank}验证，将在启动自动完成时出现。您可以在自动完成处选择合约地址。如果在自动完成时未出现合约地址，您可以输入您的合约地址，点击**+**按钮，然后点击Click To Upload按钮上传ABI
```
Comment (eshaben) [inline builders/integrations/analytics/dapplooker.md:40]: @hyd628 can you please verify this is correct?

**Review by eshaben — COMMENTED** [review]
just one thing I wasn't 100% sure about... but other than that it looks good! thanks for adding in the SEO changes too 😄

File: builders/integrations/analytics/dapplooker.md (hunk 1 above)
Comment (hyd628) [reply builders/integrations/analytics/dapplooker.md]: changed this slightly.

**Comment by hyd628** [comment]
Thanks @eshaben changed that part slightly.

**Review by eshaben — APPROVED** [review]
Thanks Henry! LGTM 🚢

**Pull request merged**